Source/Config/Main.lua

```diff
@@ -22,8 +35,16 @@ local defaults = {
   [Auctionator.Config.Options.SHOW_LISTS] = true,
   [Auctionator.Config.Options.AUTOSCAN] = true,
   [Auctionator.Config.Options.AUTO_LIST_SEARCH] = true,
-  [Auctionator.Config.Options.UNDERCUT_PERCENTAGE] = 5,
-  [Auctionator.Config.Options.AUCTION_DURATION] = 24,
+
+  [Auctionator.Config.Options.ITEM_AUCTION_DURATION] = 24,
+  [Auctionator.Config.Options.ITEM_AUCTION_SALES_PREFERENCE] = Auctionator.Config.SalesTypes.PERCENTAGE,
+  [Auctionator.Config.Options.ITEM_UNDERCUT_PERCENTAGE] = 5,
+  [Auctionator.Config.Options.ITEM_UNDERCUT_STATIC_VALUE] = 100,
+
+  [Auctionator.Config.Options.COMMODITY_AUCTION_DURATION] = 12,
```

Comment:
I think this should be 24 and items at 48, but that's just me 🤷‍♂

Reply:
Makes as much sense to me as any other setting, and its configurable so 🤷‍♂